src/js/jobs/utils.js

```diff
@@ -1,233 +1,47 @@
-        title: "Replace Previous",
-        description: "Make the new build the active index for the reference."
-    }
-};
-
-const createSampleDescriptions = {
-    make_sample_dir: {
-        title: "Create Directory",
-        description: "Create a data directory for the new sample."
-    },
-    copy_files: {
-        title: "Copy Files",
-        description: "Copy and compress input FASTQ files."
-    },
-    fastqc: {
-        title: "FastQC",
-        description: "Run FastQC on the sample FASTQ files."
-    },
-    parse_fastqc: {
-        title: "Parse FastQC",
-        description: "Parse FastQC output and store in database."
-    },
-    clean_watch: {
-        title: "Cleanup",
-        description: "Remove the source files."
-    }
-};
-
-const createSubtractionDescriptions = {
-    make_subtraction_dir: {
-        title: "Create Directory",
-        description: "Create a data directory for the new subtraction."
-    },
-    unpack: {
-        title: "Unpack",
-        description: "Decompress the sequence file if necessary."
-    },
-    set_stats: {
-        title: "Save Sequence Statistics",
-        description: "Save the GC content and sequence count to the database."
-    },
-    bowtie_build: {
-        title: "Bowtie Build",
-        description: "Build a Bowtie2 index from the FASTA file provided by the user."
-    },
-    compress: {
-        title: "Compress FASTA",
-        description: "Compress the FASTA data for long-term storage."
-    }
-};
-
-const pathoscopeBowtieDescriptions = {
-    make_analysis_dir: makeAnalysisDirDescription,
-    prepare_reads: prepareReadsDescription,
-    prepare_qc: prepareQCDescription,
-    map_default_isolates: {
-        title: "Map to Default Isolates",
-        description: "Map the sequences of all OTU default isolates."
-    },
-    generate_isolate_fasta: {
-        title: "Write Isolate FASTA",
-        description: "Create a FASTA file containing all sequences of OTUs detected in previous step."
-    },
-    build_isolate_index: {
-        title: "Build Isolate Index",
-        description: "Build a Bowtie2 index from the all-isolate FASTA generated in the previous step."
-    },
-    map_isolates: {
-        title: "Map to Isolates",
-        description: "Map to the all-isolate index."
-    },
-    map_subtraction: {
-        title: "Map to Subtraction",
-        description: "Map all reads mapped in the previous step to the subtraction."
-    },
-    subtract_mapping: {
-        title: "Subtract Reads",
-        description: "Remove reads that have a better alignment against the subtraction than an OTU."
-    },
-    pathoscope: {
-        title: "Pathoscope",
-        description: "Reassign ambiguous mappings using Pathoscope 2.0."
-    },
-    import_results: importResultsDescription,
-    cleanup_indexes: {
-        title: "Clean-up Indexes",
-        description: "Prune stale OTU indexes if they are no longer being used."
+export const getStepDescription = step => {
```

Comment:
Not strictly required, but I would consider destructuring step here. Something like:
```suggestion
export const getStepDescription = step => {
  const {state, step_name, step_description} = step;
```

Reply:
This is preferrable:
```js
export const getStepDescription = (state, name, description) => {};
```
Function signature is much better defined.